src/shapes/blender.cpp

```diff
@@ -0,0 +1,263 @@
+#include <mitsuba/render/mesh.h>
+#include <enoki/color.h>
+#include "blender_types.h"
+
+NAMESPACE_BEGIN(mitsuba)
+
+/**
+
+Blender mesh loader
+----------------------------------------------------------
+
+This plugins converts a Blender Mesh to mitsuba's mesh layout
```

Comment:
Could be more descriptive on how to use this in Python? E.g. could provide a simple code snippet that would be great!